docs/src/main/markdown/validation.md

```diff
@@ -108,6 +108,30 @@ Validator[Person].validate(personInstance) match {
 }
 ```
 
+## Package-scoped extension options
```

Comment:
this is not available outside preview, but it didn't really fit with the rest and the preview is going to be merged shortly so I thought it would be better up